src/test/java/bio/terra/integration/DataRepoFixtures.java

```diff
@@ -1885,7 +1885,7 @@ public SnapshotBuilderGetConceptsResponse getConcepts(
   }
 
   public SnapshotBuilderGetConceptsResponse searchConcepts(
-      TestConfiguration.User user, UUID datasetId, String domainId, String searchText)
+      TestConfiguration.User user, UUID datasetId, Integer domainId, String searchText)
```

Comment:
If this parameter can't be null, it should be declared as `int`, to make the intent of the code clear
```suggestion
      TestConfiguration.User user, UUID datasetId, int domainId, String searchText)
```